### PR TITLE
Remove reference to deprecated static-compiler plugin

### PR DESCRIPTION
**Pull request by joliss**
I think it might confuse people?

### CODE REVIEW

**Comment by rwjblue** [comment]
I'm ok with removing the reference if it is confusing, but the statement is accurate (it was inspired by static compiler...)

**Comment by joliss** [comment]
Let me rephrase "confusing": Now that most people aren't using static-compiler anymore (and static-compiler is deprecated), I think the reference isn't helpful to users - they just click through and see that it's deprecated. I guess the only remaining purpose is to give credit to static-compiler, but, as the author of static-compiler, I really don't care. ;)

But if you want to leave it in that's fine too.

**Comment by rwjblue** [comment]
>  I guess the only remaining purpose is to give credit to static-compiler

Confirm, this was my intent.
